src/Commands/CheckTranslationsCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace NextApps\PoeditorSync\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use NextApps\PoeditorSync\Poeditor\Poeditor;
+use NextApps\PoeditorSync\Translations\TranslationManager;
+
+class CheckTranslationsCommand extends Command
```

Comment:
```suggestion
class CheckCommand extends Command
```
or would 'poeditor:status' be better as name 🤔